graphql-dgs-codegen-core/src/test/kotlin/com/netflix/graphql/dgs/codegen/CodeGenTest.kt

```diff
@@ -1871,6 +1874,55 @@ class CodeGenTest {
             .isEqualTo("""{mandatoryString:"a",mandatoryInt:1,mandatoryList:["x"]}""")
     }
 
+    @Test
+    fun generateObjectTypeInterfaceWithInterface() {
+        val schema = """
+        type Team {
+            name: String
+        }
+
+        type Player {
+            name: String
+        }
+
+        interface Standing {
+            position: Int!
+            team: Team!
+        }
+        """.trimIndent()
+
+        val result = CodeGen(
+            CodeGenConfig(
+                schemas = setOf(schema),
+                packageName = basePackageName,
+                generateInterfaces = true
+            )
+        ).generate() as CodeGenResult
+
+        val interfaces = result.interfaces
+        val dataTypes = result.dataTypes
+
+        assertThat(interfaces).hasSize(3) // IPlayer, ITeam, Standing
```

Comment:
Instead of the comment, why not just assert the type names?

Reply:
Good point, I already do that below, will get rid of the comment.